source_gen_test/lib/src/annotations.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: comment_references
```

Comment:
get rid of the references. They aren't worth putting in if they don't work.

Reply:
with special hacks in `dartdoc` they often DO work.